docs/services/matrix.md

```diff
@@ -330,8 +326,8 @@ appservice:
 ```
 
 Unter dem Abschnitt `bridge:` befinden sich viele Konfigurationen, welche nur das Bridgen an sich betreffen. Diese müssen
-nach den persönlichen vorlieben eingestellt werden. Wir empfehlen aus den oben genannten Gründen die WhatsApp Bridge 
-nicht jedem Nutzer auf dem Homeserver zur Verfügung zu stellen, deswegen sollte man folgende Einstellung vornehmen.
+nach den persönlichen vorlieben eingestellt werden. Wir empfehlen aus den oben genannten Gründen die Bridges nicht
```

Comment:
Warum?? Die Dokumentation ist auf die WhatsApp Bridge zugeschnitten und bei den anderen Bridges, existieren keine Kollisionen mit multiuser.
Falls doch, diese separat in der Dokumentation der Bridge Erläutern.

Reply:
Stimmt prinzipiell.